scipy/stats/_levy_stable/c_src/levyst.c

```diff
@@ -14,6 +14,13 @@
 #include <stdlib.h>
 #include "levyst.h"
 
+/* M_PI et al. are not defined in math.h in C99, even with _USE_MATH_DEFINES */
```

Comment:
Interesting. Today I learned. I'll keep that in mind for the future.

https://stackoverflow.com/questions/5007925/using-m-pi-with-c89-standard